optimize/client/src/style.scss

```diff
@@ -1,14 +1,14 @@
-@use '@carbon/react' with (
-  $css--reset: false,
+@use '@carbon/react' as * with (
   $css--default-type: false,
   $font-path: 'https://fonts.camunda.io'
 ); // the 'with' part is temporary solution and has to be deleted when whole Optimize will be carbonized
 
-@use '@carbon/react/scss/fonts/sans';
 @use '@carbon/react/scss/themes';
 @use '@carbon/react/scss/theme';
-@include sans.medium();
-@include sans.bold();
```

Comment:
The bold font now is not working, i looked at the network tab and it is not being downloaded at all anymore. Number reports on dashboard are using semi bold now instead.

Reply:
okay, I see, I will bring this back then